helix-view/src/editor.rs

```diff
@@ -67,20 +72,38 @@ impl Editor {
             theme,
             language_servers,
             registers: Registers::default(),
-            status_msg: None,
+            status: String::new(),
+            severity: None,
         }
     }
 
+    /// Get immutable status. Should check severity to see if status should be empty.
+    pub fn status(&self) -> &str {
+        &self.status
+    }
+
+    pub fn severity(&self) -> &Option<Severity> {
```

Comment:
Since `Severity` is a trivial type with same size as an integer, it could be `Clone` and `Copy` and this function could simply returns `Option<Severity>`.

Reply:
Ah, I didn't check that. Yeah, `Copy` would be better.